tests/jobs/localisation/example_case/scripts/common_functions.py

```diff
@@ -0,0 +1,571 @@
+"""
+Common functions used by the scripts: init_test_case.py and sim_field.py
+"""
+import math
+from dataclasses import dataclass
+from typing import Tuple
+
+import gstools as gs
+import numpy as np
+import xtgeo
+
+# pylint: disable=missing-function-docstring, too-many-locals, invalid-name
+# pylint: disable=raise-missing-from
+# pylint: disable=too-many-nested-blocks
+
+
+# Settings for the test case in the following dataclasses
+@dataclass
+class GridSize:
+    """
+    Length, width, thickness of a box containing the field
+    Same size is used for both fine scale grid with
+    the simulated field and the coarse scale grid
+    containing upscaled values of the simulated field.
+    """
+
+    xsize: float = 7500.0
+    ysize: float = 12500.0
+    #    xsize: float = 7500.0
+    #    ysize: float = 7500.0
+    zsize: float = 50.0
+    use_eclipse_grid_index_origo: bool = False
+
+
+@dataclass
+class Field:
+    """
+    Define the dimension (number of grid cells) for fine scale grid,
+    name of output files and specification of model parameters for
+    simulation of gaussian field with option to use linear trend.
+    Relative standard deviation specify standard deviation of
+    gaussian residual field relative to the trends span of value
+    (max trend value - min trend value)
+    """
+
+    # pylint: disable=too-many-instance-attributes
+    name: str = "FIELDPARAM"
+    algorithm: str = "gstools"
+    initial_file_name: str = "init_files/FieldParam.roff"
+    updated_file_name: str = "FieldParam.roff"
+    seed_file: str = "randomseeds.txt"
+    variogram: str = "exponential"
+    correlation_range: Tuple[float] = (3000.0, 1000.0, 2.0)
+    correlation_azimuth: float = 45.0
+    correlation_dip: float = 0.0
+    correlation_exponent: float = 1.9
+    trend_use: bool = False
+    trend_params: Tuple[float] = (1.0, -1.0)
```

Comment:
I think we should get rid of these type-hints for now as `semeio` does not use `mypy` and we'll probably just end up with a bunch of wrongly typed variables.
We should definitely start using `mypy` in `semeio` but can save that for another day.